homeassistant/components/rhasspy/const.py

```diff
@@ -0,0 +1,122 @@
+"""Constants for Rhasspy integration.
+
+For more details about this integration, please refer to the documentation at
+https://home-assistant.io/integrations/rhasspy/
+"""
+DOMAIN = "rhasspy"
+
+# Language-specific profiles available for Rhasspy.
+# See https://github.com/synesthesiam/rhasspy-profiles/releases
+SUPPORT_LANGUAGES = [
+    "en-US",
+    "nl-NL",
+    "fr-FR",
+    "de-DE",
+    "el-GR",
+    "it-IT",
+    "pt-BR",
+    "ru-RU",
+    "es-ES",
+    "sv-SV",
+    "vi-VI",
+]
+
+# -----------------------------------------------------------------------------
+# Rhasspy Intents
+# -----------------------------------------------------------------------------
+
+# Confirms or disconfirms if a device is currently on
+INTENT_IS_DEVICE_ON = "IsDeviceOn"
+
+# Confirms or disconfirms if a device is currently off
+INTENT_IS_DEVICE_OFF = "IsDeviceOff"
+
+# Confirms or disconfirms if a cover is currently open
+INTENT_IS_COVER_OPEN = "IsCoverOpen"
```

Comment:
These intents should become part of Home Assistant core, they should not be part of the Rhasspy integration. That way they are available for anyone using them. 

I've opened #29280 that will make it easier to add these. Integrations can include a new `intent.py` to register them. See the PR for an example.